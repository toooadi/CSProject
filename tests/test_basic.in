GET$5$hallo
SET$5$hallo$4$welt
GET$5$hallo
