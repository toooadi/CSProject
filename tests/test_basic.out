ERR
OK
VALUE$4$welt
